examples/sensors-viewer/src/App.js

```diff
@@ -0,0 +1,20 @@
+
+import "bootstrap/dist/css/bootstrap.min.css";
+import "jquery"
+import "react-popper"
+import './assets/css/cast.css'
+
+import React, {Component} from 'react';
+import SensorViewer from "./components/SensorViewer";
+
+class App extends Component {
```

Comment:
This class is almost empty. You can Initialize reacr directly with SensorViewer or move the "app" logic of SensorViewer here.